demo/index.html

```diff
@@ -27,105 +30,136 @@
     <div class="c-tab c-tab--block c-tab--main">
       <ul class="c-tab__list c-tab__list">
         <li class="c-tab__list__item" tabindex="0">
-          <a href="//zendeskgarden.github.io/" tabindex="-1">Home</a>
+          <a href="/" tabindex="-1">Home</a>
         </li>
         <li class="c-tab__list__item" tabindex="0">
-          <a href="//zendeskgarden.github.io/assets/" tabindex="-1">Assets</a>
+          <a href="/assets" tabindex="-1">Assets</a>
         </li>
         <li class="c-tab__list__item" tabindex="0">
-          <a href="//zendeskgarden.github.io/css-components/" tabindex="-1">CSS Components</a>
+          <a href="//zendeskgarden.github.io/css-components" tabindex="-1">CSS Components</a>
         </li>
         <li class="c-tab__list__item is-selected" tabindex="0">
-          <a href="/react-components" tabindex="-1">React Components</a>
+          <a href="//zendeskgarden.github.io/react-components/" tabindex="-1">React Components</a>
         </li>
       </ul>
     </div>
   </nav>
+  <header class="c-header c-main__header">
+    <figure class="c-header__logo">
+      <a class="u-fg-inherit" href="/">
+        <svg class="c-header__logo__svg svg-square">
+          <use xlink:href="index.svg#zd-svg-icon-26-zendesk">
+        </svg>
+      </a
+    ></figure
+    >
+    <ul class="c-ctl c-ctl--header">
+      <li class="c-ctl__item u-position-relative">
+        <a class="c-menu-ctl c-header__menu js-menu" href="#settings">
+          &nbsp;
+          <svg>
+            <use xlink:href="index.svg#zd-svg-icon-16-menu-stroke">
+          </svg>
+        </a>
+        <ul aria-hidden="true" class="c-menu c-menu--down c-menu--sm" role="menu">
+          <a href="/" class="c-menu__item" role="menuitem">
+            Home
+          </a>
```

Comment:
Remove the "Home" menu item here, as the linked "Z" covers that nav option on mobile.

Reply:
Removed.